##########################################################################################
##########################################################################################
################################编队控制器外部参数##########################################
##########################################################################################
##########################################################################################

kv_p: 0.5 #主从机速度差比例项

kp_p: 0.8 #从机期望与实际位置误差比例

mix_kp: 0.6 #总混合产生期望空速pid参数

mix_kd: 0.0 #总混合产生期望空速pid参数

mix_ki: 0.01 #总混合产生期望空速pid参数

maxinc_acc: 5.0 #飞机前向最大瞬时加速度

maxdec_acc: 3.0 #飞机减速最大瞬时加速度

max_arispd_sp: 25.0 #飞机空速最大设定值,此处的最大速度，一定要和飞机的最快速度贴合，否则容易造成油门抖动

min_arispd_sp: 8.0 #飞机空速最小设定值

##########################################################################################
##########################################################################################
################################编队控制器内部TECS参数######################################
##########################################################################################
##########################################################################################

EAS2TAS : 1

climboutdem : false 

climbout_pitch_min_rad : 0.2 

throttle_min : 0.1 

throttle_max : 1 

throttle_cruise : 0.1 

pitch_min_rad : -0.5 

pitch_max_rad : 0.5 

speed_weight : 1 

time_const_throt : 8.0 

time_const : 5.0 

##########################################################################################
##########################################################################################
################################编队控制器内部横侧向参数####################################
##########################################################################################
##########################################################################################

roll_max: PI/2